packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -987,32 +987,32 @@ export function getElementPadding(element) {
 
 /**
  * Returns the absolute height of an element.
+ * Uses getBoundingClientRect() to ensure synchronous layout calculation.
  * @param {Element} element
  * @returns {number} The absolute height of the element including padding.
  */
 export function getAbsoluteHeight(element) {
   if (!element) {
     return 0;
   }
-  const paddings = getElementPadding(element);
-  const padding = paddings.top + paddings.bottom;
 
-  // @ts-ignore
-  return Math.ceil(element.offsetHeight + padding);
+  // Using getBoundingClientRect ensures accurate height measurements across all browsers, especially Safari.
+  const elementBoundingRect = element.getBoundingClientRect();
+  return Math.ceil(elementBoundingRect.height);
```

Comment:
This method is used in other places than the CRGA component, you will need to make sure it still works elsewhere as well if you change it.

It is used in the QuanticTooltip, and in the QuanticSmartSnippet. Please make sure you visually test those components also.